src/config/sponsors.js

```diff
@@ -0,0 +1,56 @@
+export default [
+  {
+    name: 'TEKsystems',
+    desc: 'Whether it’s transforming your business, building an all-star team or finding the right fit for your professional ambitions, we’re in it with you.',
+    logo: 'teksystems.png',
+    url: 'https://www.teksystems.com/'
+  },
+  {
+    name: 'RMC Agency',
+    desc: 'A Michigan based, full-service Information Technology (IT) Consulting and Placement agency.',
+    logo: 'RMC.png',
+    url: 'http://www.rmcagency.com/'
+  },
+  {
+    name: 'SBDC Lansing',
+    desc: 'The Michigan Small Business Development Center offers expert assistance at no cost to entrepreneurs looking to start or grow a business.',
+    logo: 'SBDC_MI.png',
+    url: 'https://sbdcmichigan.org/'
+  },
+  {
+    name: 'A2 Hosting',
+    desc: 'We have a blazing fast web hosting solution perfect for you! Whether you have a low traffic, brand new blog or a popular business site, we\'ve got you covered! ',
+    logo: 'a2_hosting.png',
+    url: 'https://www.a2hosting.com/'
+  },
+  {
+    name: 'LEAP',
+    desc: 'This is a group for UX design practitioners and those interested in learning about design.',
+    logo: 'leap.png',
+    url: 'http://www.purelansing.com/'
+  },
+  {
+    name: 'Humanity Codes',
+    desc: 'Humanity Codes uses code and education to bring together Lansing\'s talented and diverse tech community.',
+    logo: 'lansing-codes.png',
+    url: 'https://www.lansing.codes/'
+  },
+  {
+    name: 'BrightDigit',
+    desc: 'The BrightDigit team brings over 15 years of experience in technology. Together they have a passion for mobile technology, public speaking, and education.',
+    logo: 'bright_digit.png',
+    url: 'https://brightdigit.com/'
+  },
+  {
+    name: 'TechSmith',
+    desc: 'Founded in 1987, TechSmith pioneered the revolutionary idea of capturing screen content for better communication. Today, TechSmith is the world’s #1 source for visual communication software.',
+    logo: 'tech_smith.png',
+    url: 'https://www.techsmith.com/'
+  },
+  {
+    name: 'Kunz, Leigh, and Associates',
+    desc: 'Kunz, Leigh and Associates is a software development and professional services company currently serving the private and public sectors in Michigan and Ohio.',
+    logo: 'KLA.png',
+    url: 'https://kunzleigh.com/'
+  }
```

Comment:
Sorry I didn't catch this the first time. Would you mind adding Hyperverses as well? The link is on #9. @pfhawkins can provide a good image and description. 🙂

Reply:
I have updated Hyperverses! This should be all set.